api/v1alpha1/managedcluster_types.go

```diff
@@ -60,6 +62,45 @@ const (
 	ProgressingReason string = "Progressing"
 )
 
+// ManagedClusterServiceSpec represents a Service within ManagedCluster
+type ManagedClusterServiceSpec struct {
+	// Template is a reference to a Template object located in the same namespace.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	Template string `json:"template"`
+	// Install tells if this service should be installed.
+	// +kubebuilder:default:=true
+	Install bool `json:"install"`
```

Comment:
I'd suggest `Disable` instead. As it not only controls installation, but also removes the service from the cluster. It's also inverse, so will be false by default and hidden with ommitempty.